### ISSUE TITLE
Network architecture details

### ISSUE THREAD

**Issue by ntwong0** [SJSURoboticsTeam/urc-autonomy-subsystem-2021#6]
## To Do
- Layers for the neural network

Earlier layers: Transfer learning from ResNet
Later Layers: ( ( CONV + RELU +  BATCH + DROPOUT) * N + POOL) * M

- Things we want to classify
- What is the goal

## How to Test?

- GPU
- Real-time deep learning

##Ideas for v2

- Add capabilities to detect texture

**Comment by ntwong0** [comment]
Mar 10 at 3:31 PM
model and weights to be uploaded to https://drive.google.com/drive/folders/1s-0zQVtm-LDr0RJCGAZHoI5iUVmJJDd_

Mar 10 at 3:29 PM
Documentation in progress, with directions on how to set up the model and weights on another computer

Mar 10 at 3:27 PM
model selected
network trained
